Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -289,6 +289,28 @@ var ListView = React.createClass({
     }
   },
 
+  /**
+   * If this is a vertical ListView scrolls to the bottom.
+   * If this is a horizontal ListView scrolls to the right.
+   *
+   * Use `scrollToEnd()` for immediate scrolling,
+   * `scrollToEnd({animated: true})` for animated scrolling.
+   *
+   * See `ScrollView#scrollToEnd`.
+   */
+  scrollToEnd: function(options?: { animated?: boolean }) {
```

Comment:
I like that the user can call `scrollTo` and `scrollToEnd` directly on the `ListView` though, without having to reach into its internal scroll component. It also encapsulates the functionality and we could change the implementation of these methods without breaking clients in the future.